mlir/include/mlir/Dialect/Tosa/IR/TosaComplianceData.h.inc

```diff
@@ -406,9 +431,9 @@ extensionComplianceMap = {
       {{Extension::fp8e4m3}, {{fp8e4m3T, fp8e4m3T}}},
       {{Extension::fp8e5m2}, {{fp8e5m2T, fp8e5m2T}}},
       {{Extension::bf16}, {{bf16T, bf16T}}}}},
-    {"tosa.dim",
-     {{{Extension::fp8e4m3}, {{fp8e4m3T}}},
-      {{Extension::fp8e5m2}, {{fp8e5m2T}}},
-      {{Extension::bf16}, {{bf16T}}}}},
+    {"tosa.variable_write",
```

Comment:
Same here

Reply:
Also couldn't see `tosa.cond_if` and `tosa.while_loop`, should they be listed here since they are dependent on `EXT-CONTROLFLOW`?

Reply:
The supported extension of tosa.cond_if and tosa.while_loop is listed in the availability section. These ops support all types (based on `Supported Data Types` section of the spec) , so no needed to be listed here.

Reply:
Add profile validation check on the type of condition output for cond_if and while_loop.